Makefile

```diff
@@ -32,6 +32,14 @@ clean: ## Removes binary, cache folder and docker images
 	@rm -rf ${BINARY_DIR}
 	@rm -rf $(PLUGIN_DEPLOYMENTS_DIR)
 
+.PHONY: tools
+tools: ## Installs required go tools
+	@go get -u github.com/alecthomas/gometalinter
+	@go get -u github.com/alecthomas/gometalinter && gometalinter --install
```

Comment:
why the getting of gometalinter `@go get -u github.com/alecthomas/gometalinter` is called twice?

Reply:
Copy-pastie :) Somehow missed that. Fix on the way

Reply:
Fixed with d7a909f.